crates/astria-sequencer/src/ibc/ics20_transfer.rs

```diff
@@ -424,7 +413,7 @@ async fn execute_ics20_transfer<S: StateWriteExt>(
     // the IBC packet should have the address as a bech32 string
     let recipient = Address::try_from_bech32m(&recipient).context("invalid recipient address")?;
 
-    let is_prefixed = is_prefixed(source_port, source_channel, &unprefixed_denom);
+    let is_prefixed = denom_trace.has_exact_path(&format!("{source_port}/{source_channel}"));
```

Comment:
As above, is the exact match maybe too strict? Would it be enough to check if it just starts with `{source_port}/{source_channel}`?

Reply:
yeah, should just check the start

Reply:
As above, replaced `TracePrefixed::has_exact_path` replaced by `TracePrefixed::starts_with_str`  in 010b7e